web/src/app/modules/views/main/editors/modules/graphical-editor/components/graphical-editor.component.ts

```diff
@@ -187,6 +187,32 @@ export class GraphicalEditor {
                 this.graph.getSelectionModel().addCells(this.graph.getChildCells(this.graph.getDefaultParent()));
             }
         });
+
+        // Ctrl+C
+        this.keyHandler.bindControlKey(67, (evt: KeyboardEvent) => {
+          this.stopEvent(evt);
+          if (this.graph.isEnabled()) {
+            mx.mxClipboard.copy(this.graph, this.graph.getSelectionCells());
+          }
+        });
+
+        // Ctrl+V
+        this.keyHandler.bindControlKey(86, (evt: KeyboardEvent) => {
+          this.stopEvent(evt);
+          if (this.graph.isEnabled()) {
+            mx.mxClipboard.paste(this.graph);
+          }
+        });
+
+        // Ctrl+X
+        this.keyHandler.bindControlKey(88, (evt: KeyboardEvent) => {
+          this.stopEvent(evt);
+          if (this.graph.isEnabled()) {
+            mx.mxClipboard.copy(this.graph, this.graph.getSelectionCells());
+            this.deleteSelectedCells();
+
+          }
+        });

```

Comment:
Add this to support command button when using a Mac Keyboard

`        this.keyHandler.getFunction = function(evt){
            if (evt != null) {
                return (mx.mxEvent.isControlDown(evt) || (mx.mxClient.IS_MAC && evt.metaKey)) ? this.controlKeys[evt.keyCode] : this.normalKeys[evt.keyCode];
            }
        return null;
        };`

Reply:
I'm not sure if I added it in the right way. Can't test it on Windows